docker-compose/tomcat-manager/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+version: '3.3'
+services:
+  postgres:
+    image: 'postgres:9'
+    container_name: postgres
+#    ports:
+#      - '5432:5432'
+    environment:
+      - 'POSTGRES_MULTIPLE_DATABASES=apiman,${POSTGRES_KEYCLOAK_DATABASE}'
+      - 'POSTGRES_USER=${POSTGRES_USER}'
+      - 'POSTGRES_PASSWORD=${POSTGRES_PASSWORD}'
+    volumes:
+      - type: bind
+        source: $PWD/../common/create-multiple-postgresql-databases.sh
+        target: /docker-entrypoint-initdb.d/create-multiple-postgresql-databases.sh
+      - type: volume
+        source: postgres_data
+        target: /var/lib/postgres
+      - type: volume
+        source: postgres_logs
+        target: /var/log/postgres
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+#    labels:
+#      - traefik.http.routers.postgres-router.tls=true
+#      - traefik.http.routers.postgres-router.entrypoints=https
+#      - traefik.http.routers.postgres-router.rule=PathPrefix(`/postgres`)
+#      - traefik.http.routers.postgres-router.service=postgres-service
+#      - traefik.http.services.postgres-service.loadbalancer.server.port=5432
+#      - traefik.http.routers.postgres-router.middlewares=test-stripprefix
```

Comment:
test-stripprefix? guess this is copied from docs and we probably want to put a more appropriate value